js/screens/authentication/SpidInformationRequestScreen.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import * as React from 'react'
+import { connect } from 'react-redux'
+import { isValid } from 'redux-form'
+import {
+  type NavigationScreenProp,
+  type NavigationState
+} from 'react-navigation'
+import {
+  Container,
+  Content,
+  Text,
+  View,
+  Button,
+  Icon,
+  Header,
+  Left,
+  Body,
+  H1
+} from 'native-base'
+
+import { type GlobalState } from '../../reducers/types'
+import I18n from '../../i18n'
+import Modal from '../../components/ui/Modal'
+import SpidInformationForm, {
+  FORM_NAME
+} from '../../components/forms/SpidInformationForm'
+
+type ReduxMappedProps = {
+  isFormValid: boolean
+}
+
+type OwnProps = {
+  navigation: NavigationScreenProp<NavigationState>
+}
+
+type Props = ReduxMappedProps & OwnProps
+
+type State = {
+  isTosModalVisible: boolean
+}
+
+/**
+ * A screen where the user can insert an email to receive information about SPID.
+ */
+class SpidInformationRequestScreen extends React.Component<Props, State> {
+  state: State = {
```

Comment:
also can you please check why Flow didn't catch that `isFormValid` is missing from `State` ?

Reply:
Read above